src/isEqual/index.ts

```diff
@@ -26,10 +26,10 @@ import requiredArgs from '../_lib/requiredArgs/index'
  * )
  * //=> false
  */
-export default function isEqual(dirtyLeftDate, dirtyRightDate) {
+export default function isEqual(dirtyLeftDate: Date | number, dirtyRightDate: Date | number): boolean {
   requiredArgs(2, arguments)
 
-  var dateLeft = toDate(dirtyLeftDate)
-  var dateRight = toDate(dirtyRightDate)
+  const dateLeft: Date = toDate(dirtyLeftDate)
+  const dateRight: Date = toDate(dirtyRightDate)
```

Comment:
I don't think it's required, could you please confirm?

```suggestion
  const dateLeft = toDate(dirtyLeftDate)
  const dateRight = toDate(dirtyRightDate)
```